examples/demo-rollup/Cargo.toml

```diff
@@ -72,13 +72,14 @@ ethers = { workspace = true }
 revm = { workspace = true }
 
 [features]
-default = ["native"] # Deviate from convention by making the "native" feature active by default. This aligns with how this package is meant to be used (as a binary first, library second).
+default = ["native", "experimental"] # Deviate from convention by making the "native" feature active by default. This aligns with how this package is meant to be used (as a binary first, library second).
 experimental = ["sov-ethereum/experimental", "reth-primitives", "secp256k1", "demo-stf/experimental"]
 native = ["anyhow", "jsonrpsee", "serde", "serde_json", "tracing", "tokio", "tracing-subscriber",
 	"demo-stf/native", "sov-modules-stf-template/native", "sov-risc0-adapter/native", "sov-modules-api/native",
 	"sov-state/native", "sov-cli", "clap", "sov-celestia-adapter/native", "sov-db", "sov-sequencer", "sov-stf-runner/native",
-	"sov-modules-api/native"]
+	"sov-modules-api/native", "local"]
```

Comment:
As it is part of the evm work let's add `local` to `experimental` instead of `native`.